pkg/scheduler/testing/wrappers.go

```diff
@@ -417,6 +446,20 @@ func (p *PodWrapper) Label(k, v string) *PodWrapper {
 	return p
 }
 
+// MetadataLabels sets all {k,v} pair provided by `labels` to the inner pod.
+func (p *PodWrapper) MetadataLabels(labels map[string]string) *PodWrapper {
```

Comment:
Given `Label` method is exist, we should name this as `Labels`?

Reply:
`Labels` is a Pod attribute so cannot reuse that as a function or it will conflict.

Reply:
Really? I think we can because we already have `Name` method on PodWrapper.

Reply:
> Labels is a Pod attribute so cannot reuse that as a function or it will conflict.

We can actually :) In that case we should specify the full path rather than emitting the anonymous field (ObjectMeta):

```go
// Label sets a {k,v} pair to the inner pod.
func (p *PodWrapper) Label(k, v string) *PodWrapper {
	if p.ObjectMeta.Labels == nil {
		p.ObjectMeta.Labels = make(map[string]string)
	}
	p.ObjectMeta.Labels[k] = v
	return p
}

// Labels sets all {k,v} pair provided by `labels` to the inner pod.
func (p *PodWrapper) Labels(labels map[string]string) *PodWrapper {
	for k, v := range labels {
		p.Label(k, v)
	}
	return p
}
```

Reply:
Got it, thanks!
Fixed.